schemas/com.snowplowanalytics.snowplow.enrichments/api_request_enrichment_config/jsonschema/1-0-2

```diff
@@ -0,0 +1,169 @@
+								},
+								"additionalProperties": false
+							},
+							"json": {
+								"type": "object",
+								"properties": {
+									"field": {
+										"type": "string",
+										"enum": ["unstruct_event", "contexts", "derived_contexts"]
+									},
+									"schemaCriterion": {
+										"type": "string",
+										"pattern": "^iglu:[a-zA-Z0-9-_.]+/[a-zA-Z0-9-_]+/[a-zA-Z0-9-_]+/([1-9][0-9]*|\\*)-((?:0|[1-9][0-9]*)|\\*)-((?:0|[1-9][0-9]*)|\\*)$"
+									},
+									"jsonPath": {
+										"type": "string",
+										"pattern": "^\\$.*$"
+									}
+								},
+								"additionalProperties": false
+							}
+						},
+						"additionalProperties": false,
+						"minProperties": 2,
+						"maxProperties": 2,
+						"required": ["key"]
+					}
+				},
+				"api": {
+					"type": "object",
+					"minProperties": 1,
+					"maxProperties": 1,
+					"properties": {
+						"http": {
+							"type": "object",
+							"properties": {
+								"method": {
+									"type": "string",
+									"enum": ["GET", "POST", "PUT"]
+								},
+								"uri": {
+									"type": "string"
+								},
+								"timeout": {
+									"type": "integer",
+									"minimum": 1,
+									"maximum": 60000
+								},
+								"authentication": {
+									"type": "object",
+									"properties": {
+										"httpBasic": {
+											"type": "object",
+											"properties": {
+												"username": {
+													"type": "string"
+												},
+												"password": {
+													"type": "string"
+												}
+											},
+											"required": ["username", "password"],
+											"additionalProperties": false
+										}
+									},
+									"additionalProperties": false
+								}
+							},
+							"required": ["method", "uri", "timeout", "authentication"],
+							"additionalProperties": false
+						}
+					},
+					"additionalProperties": false
+				},
+				"outputs": {
+					"type": "array",
+					"minItems": 1,
+					"items": {
+						"type": "object",
+						"properties": {
+							"schema": {
+								"type": "string",
+								"pattern": "^iglu:([a-zA-Z0-9-_.]+)/([a-zA-Z0-9-_]+)/([a-zA-Z0-9-_]+)/([1-9][0-9]*(?:-(?:0|[1-9][0-9]*)){2})$"
+							},
+							"json": {
+								"type": "object",
+								"properties": {
+									"jsonPath": {
+										"type": "string",
+										"pattern": "^\\$.*$"
+									}
+								},
+								"required": ["jsonPath"],
+								"additionalProperties": false
+							}
+						},
+						"required": ["schema"],
+						"minProperties": 2,
+						"maxProperties": 2,
+						"additionalProperties": false
+					}
+				},
+				"cache": {
+					"type": "object",
+					"properties": {
+						"size": {
+							"type": "integer",
+							"minimum": 0
+						},
+						"ttl": {
+							"type": "integer",
+							"minimum": 0,
+							"maximum": 86400
+						}
+					},
+					"additionalProperties": false,
+					"required": ["size", "ttl"]
+				},
+				"ignoreOnError": {
+					"type": "boolean"
```

Comment:
Not to nitpick - should this be nullable if this is not required?

Reply:
So now we could omit `ignoreOnError` but not set explicitly `null`. With `"type": ["boolean", "null"]` we could omit and also set `null`. I can't see the reason why we shoudn't do that, so...I will add `null` ;)